tests/compare_manifests.sh

```diff
@@ -16,15 +16,15 @@ sed -i.bak '/caBundle:/d' helm_operator_templates.yaml
 sed -i.bak '/caBundle:/d' helm_chart_templates.yaml
 
 # Remove line with tag in Prometheus Spec
-sed -i.bak '/tag: v2.22.1/d' helm_operator_templates.yaml
+#sed -i.bak '/tag: v2.22.1/d' helm_operator_templates.yaml
```

Comment:
Should these remain commented out?

Reply:
They have no validity anymore since those replacements are not needed. I wanted to have a second opinion before I get rid of the lines, hence commented out.